From e384ae2dd6a2e157788f325f3e4fdf92349daec5 Mon Sep 17 00:00:00 2001
From: Steve Kowalik <steven@wedontsleep.org>
Date: Wed, 6 Sep 2023 12:52:41 +1000
Subject: [PATCH] Remove future requirement

Now that we support Python >= 3.7, we can remove the requirement on
the future module, which appears to be unused.
---
 requirements.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/requirements.txt b/requirements.txt
index 57ecd72eb..3359b30a6 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -2,7 +2,6 @@ setuptools>=38.4.0
 cffi>=1.11.3
 paramiko>=2.6.0
 requests>=2.7.0
-future
 textfsm
 jinja2
 netaddr
